### PR TITLE
Added section about MeerKATHI and updated docs on IDIA pipeline

### PR DESCRIPTION
**Pull request by Jordatious**

### CODE REVIEW

File: docs/astronomy/astronomy_software.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ## Astronomy Software
 
-### IDIA Pipelines
+### IDIA Pipeline
 
-The IDIA Pipelines software has been developed as a tool for calibration and imaging of radio data using the ilifu system. Please visit the IDIA Pipeline [website](https://idia-pipelines.github.io/) for additional information, including documentation and tutorials.
+The IDIA `processMeerKAT` Pipeline has been developed as a tool for calibration and imaging of radio data using the ilifu system. It is currently a 1GC (cross-calibration) pipeline that runs on ilifu using CASA and SLURM, which is actively being extended to 2GC (self-calibration), as well as being optimised to run more than an order of magnitude faster, with good success so far. Please visit the IDIA Pipeline [website](https://idia-pipelines.github.io/) for additional information, including documentation and tutorials. We strongly encourage you to engage with the developers by reporting any bugs or enhancements to the [GitHub issues page](https://github.com/idia-astro/pipelines/issues).
```
Comment (jeremysmith123) [inline docs/astronomy/astronomy_software.md]: enhancement requests

File: docs/astronomy/astronomy_software.md [hunk 2]
```diff
@@ -33,6 +33,19 @@ srun /carta_share/hdf_convert/run_hdf_converter image.fits /carta_share/users/<u
...
+
+#### Installing MeerKATHI on ilifu
+
+Installing a stable version of MeerKATHI on ilifu is a work in progress, as captured in [this](https://github.com/ska-sa/meerkathi/issues/625) GitHub issue. Please consult this issue for the latest progress, and comment here if you make some of your own progress. Please do not install MeerKATHI in your `/users/` directory (see directory structure documentation [here](/data/directory_structure)), particularly the singularity containers.
+
+#### Running MeerKATHI on ilifu
+
+Please don't run MeerKATHI on the ilifu login node, but on a compute node, using `sbatch`, `srun`, or `salloc`, as documented [here](/getting_started/submit_job_slurm). If you encouter issues with running MeerKATHI, please consider commenting on the [GitHub issue](https://github.com/ska-sa/meerkathi/issues/625), or logging a new one, rather than contacting ilifu support, unless the issue is clearly an ilifu issue.
```
Comment (jeremysmith123) [inline docs/astronomy/astronomy_software.md]: encouter -> encounter

File: docs/astronomy/astronomy_software.md (hunk 1 above)
Comment (Jordatious) [reply docs/astronomy/astronomy_software.md]: All good?

File: docs/astronomy/astronomy_software.md (hunk 2 above)
Comment (Jordatious) [reply docs/astronomy/astronomy_software.md]: Fixed

**Pull request merged**